noxfile.py

```diff
@@ -0,0 +1,53 @@
+import os
+import platform
+import re
+import sys
+
+import nox
+
+borg_version = os.getenv("BORG_VERSION")
+
+if borg_version:
+    # Use specified borg version
+    supported_borgbackup_versions = [borg_version]
+else:
+    # Generate a list of borg versions compatible with system installed python version
+    system_python_version = platform.python_version_tuple()
```

Comment:
Why don't you use `sys.version_info`?

Reply:
I didn't know this returns version info as integer. Used it now.